apps/api/src/server/applications/application/application.controller.js

```diff
@@ -0,0 +1,57 @@
+import { filter, map } from 'lodash-es';
+import * as caseRepository from '../../repositories/case.repository.js';
+import { mapCreateApplicationRequestToRepository } from './application.mapper.js';
+
+/**
+ *
+ * @param {import('@pins/api').Schema.ServiceCustomer[] | undefined} serviceCustomers
+ * @returns {number[]}
+ */
+const getServiceCustomerIds = (serviceCustomers) => {
+	return map(serviceCustomers, (serviceCustomer) => {
+		return serviceCustomer.id;
+	});
+};
+
+/**
+ * @type {import('express').RequestHandler}
+ */
+export const createApplication = async (request, response) => {
+	const mappedApplicationDetails = mapCreateApplicationRequestToRepository(request.body);
+
+	const application = await caseRepository.createApplication(mappedApplicationDetails);
+
+	const applicantIds = getServiceCustomerIds(application.serviceCustomer);
```

Comment:
so this means that you're sending me back an array of applicant ids and (at least for now) I'll just have to pick the first one and send it back, right?

Reply:
Yep